crates/bevy_state/src/state_scoped.rs

```diff
@@ -52,7 +52,9 @@ use crate::state::{StateTransitionEvent, States};
 /// app.enable_state_scoped_entities::<GameState>();
 /// app.add_systems(OnEnter(GameState::InGame), spawn_player);
 /// ```
-#[derive(Component)]
+#[derive(Component, Clone)]
+#[cfg_attr(feature = "reflect", derive(bevy_reflect::Reflect))]
+#[cfg_attr(feature = "reflect", reflect(Component))]
```

Comment:
Sorry, that's the correct feature name, we have some inconsistency between crates :/
```suggestion
#[cfg_attr(feature = "bevy_reflect", derive(bevy_reflect::Reflect))]
#[cfg_attr(feature = "bevy_reflect", reflect(Component))]
```